config/index.ts

```diff
@@ -1,4 +1,4 @@
-const config = {
+export const config = {
```

Comment:
let's use the correct casing for global constants.

```suggestion
export const CONFIG = {
```

Reply:
Done!